### PR TITLE
Updates in packaging v2

### PR DESCRIPTION
**Pull request by sergiomb2**
Patch1 
 - Fix mix tabs and spaces with vim :retab

Patch2 
- unbundle several jars not all
- ant just linux builds
- unzip dist/*.zip as files to install, is all we want to install
- use own png for .desktop and use own startup script just do a symbol link to
  /usr/bin it works superb
- we have 64 bits, 32 bits and generic builds, generic build seems to be noarch
  build, but others are arched ... , TODO mark package as noarch for generic build.
  Also as side effect of arched builds we need add %global debug_package %{nil},
  because build ends with Empty %files file
  /builddir/build/BUILD/dbgl-0.80/debugfiles.list

Patch3 
- Use external .desktop file easier to send to upstream
- Add Packaging:AppData

### CODE REVIEW

File: dbgl.spec [hunk 1]
```diff
@@ -35,52 +46,83 @@ the product is somewhat usable as it is. Please bear in mind that
...
 install -dm 755 %{buildroot}%{_javadir}/%{name}/
-copy_files="captures db  dbgl.jar  dosroot export lib profiles templates xsl"
-for files in $copy_files
-do 
-cp -af $files %{buildroot}/%{_javadir}/%{name}/
-done
+%ifarch x86_64
+    tar xvf dist/dbgl080_64bit.tar.gz -C %{buildroot}/%{_javadir}/%{name}/
+%else
+    %ifarch i686
```
Comment (igor-raits) [inline dbgl.spec:77]: probably you meant `%{ix86}`

File: dbgl.spec [hunk 2]
```diff
@@ -35,52 +46,83 @@ the product is somewhat usable as it is. Please bear in mind that
...
-EOF
-
-desktop-file-validate %{buildroot}%{_datadir}/applications/%{name}.desktop
+desktop-file-install                               \
+--dir=%{buildroot}%{_datadir}/applications         \
+%{SOURCE1}
+
+install -D -p -m 0644 %{SOURCE2} \
+    %{buildroot}%{_datadir}/appdata/%{name}.appdata.xml
+appstream-util validate-relax --nonet \
```
Comment (igor-raits) [inline dbgl.spec:110]: to %check please

**Review by igor-raits — APPROVED** [review]
looks good. btw, you can remove `/` between `%{buildroot}` and others.

**Comment by Tigro** [comment]
@sergiomb2 Can you try these builds: 

http://koji.russianfedora.pro/koji/buildinfo?buildID=216 or
http://koji.russianfedora.pro/koji/buildinfo?buildID=217

File: dbgl.spec (hunk 1 above)
Comment (sergiomb2) [reply dbgl.spec:77]: yes , %{ix86} is more correct  and also I will remove / between %{buildroot} and others.

File: dbgl.spec (hunk 2 above)
Comment (sergiomb2) [reply dbgl.spec:110]: OK , not mandatory may stay on %install , 

Thanks

**Comment by sergiomb2** [comment]
I will thanks.

**Pull request merged**